package.json

```diff
@@ -49,6 +49,7 @@
     "eslint": "^8.23.0",
     "http-server": "^14.1.1",
     "lit-html": "^2.3.1",
+    "postcss-import": "^12.0.0",
```

Comment:
would be nice to get this "back" from Greenwood again 😅 

however, because Storybook only like `^12.x`, we might need to apply our own override.